runtime/src/main/java/org/corfudb/runtime/exceptions/WriteTooLarge.java

```diff
@@ -0,0 +1,8 @@
+package org.corfudb.runtime.exceptions;
+
+public class WriteTooLarge extends RuntimeException {
```

Comment:
Needs to end in Exception to conform to style

Reply:
Done.